src/Microsoft.Azure.WebJobs.Host/Loggers/FunctionInstanceLogger.cs

```diff
@@ -15,6 +14,7 @@ namespace Microsoft.Azure.WebJobs.Host.Loggers
 {
     internal class FunctionInstanceLogger : IFunctionInstanceLogger
     {
+        private readonly ConcurrentDictionary<string, ILogger> _cachedLoggers = new ConcurrentDictionary<string, ILogger>();
```

Comment:
We don't actually need this (I called it out in the other PR too) -- the logger factory already caches these so it's better just to use their implementation. I think some of the tests are failing because of this, too.

Reply:
Removed